frontend/src/app/app.routes.ts

```diff
@@ -153,6 +154,10 @@ export const routes: Routes = [
             }
           }
         ]
+      },
+      {
+        path: 'cities',
+        component: CitiesUploadComponent
```

Comment:
The path is accessible, but the upload is not (or shouldn't be, at least). So any upload attempt by a non-admin should fail. It's the same for all the other admin-only pages, BTW.